dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -139,4 +159,100 @@ public CollectionResource wrapResource(CollectionRest collection, String... rels
         return new CollectionResource(collection, utils, rels);
     }
 
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
+    protected CollectionRest createAndReturn(Context context) throws AuthorizeException {
+        HttpServletRequest req = getRequestService().getCurrentRequest().getHttpServletRequest();
+        ObjectMapper mapper = new ObjectMapper();
+        CollectionRest collectionRest;
+        try {
+            ServletInputStream input = req.getInputStream();
+            collectionRest = mapper.readValue(input, CollectionRest.class);
+        } catch (IOException e1) {
+            throw new UnprocessableEntityException("Error parsing request body: " + e1.toString());
+        }
+
+        Collection collection;
+
+
+        try {
+            Community parent = null;
+            if (StringUtils.isNotBlank(req.getParameter("parentCommunity"))) {
+
+                UUID owningCommunityUuid = UUIDUtils.fromString(req.getParameter("parentCommunity"));
+                if (owningCommunityUuid == null) {
+                    throw new BadRequestException("The given owningCommunityUuid was invalid: "
```

Comment:
Again, "owningCommunity" isn't a concept in DSpace.  These variables (in this method) and this error should be renamed to say something like "parentCommunity" instead.